src/Core/src/Platform/iOS/TransformationExtensions.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using CoreAnimation;
+using CoreGraphics;
+using Microsoft.Maui.Graphics;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		public static void UpdateTransformation(this UIView nativeView, IView? view)
+		{
+			CALayer? layer = nativeView.Layer;
+			bool isInteractive = nativeView.UserInteractionEnabled;
+			CGPoint? originalAnchor = layer?.AnchorPoint;
+			Rectangle? lastBounds = view?.Frame;
+
+			nativeView.UpdateTransformation(view, layer, isInteractive, originalAnchor, lastBounds);
+		}
+
+		public static void UpdateTransformation(this UIView nativeView, IView? view, CALayer? layer, bool isInteractive, CGPoint? originalAnchor, Rectangle? lastBounds)
+		{
+			if (view == null)
+				return;
+
+			bool shouldInteract;
+
+			if (view is ILayout layout)
+				shouldInteract = layout.IsEnabled;
+			else
+				shouldInteract = view.IsEnabled;
+
+			if (isInteractive != shouldInteract)
+			{
+				nativeView.UserInteractionEnabled = shouldInteract;
+				isInteractive = shouldInteract;
+			}
+
+			var boundsChanged = lastBounds != view.Frame;
+
+			var anchorX = (float)view.AnchorX;
+			var anchorY = (float)view.AnchorY;
+			var translationX = (float)view.TranslationX;
+			var translationY = (float)view.TranslationY;
+			var rotationX = (float)view.RotationX;
+			var rotationY = (float)view.RotationY;
+			var rotation = (float)view.Rotation;
+			var scale = (float)view.Scale;
+			var scaleX = (float)view.ScaleX * scale;
+			var scaleY = (float)view.ScaleY * scale;
+			var width = (float)view.Frame.Width;
+			var height = (float)view.Frame.Height;
+			var x = (float)view.Frame.X;
+			var y = (float)view.Frame.Y;
+
+			// TODO: Port Opacity and IsVisible properties.
+			var opacity = 1.0d;
+			var isVisible = true;
+
+			void Update()
+			{
+				var parent = view.Parent;
```

Comment:
Should this be the virtual view parent or the native view parent? If it is the virtual view, then a page will still have a parent - the window - and thus the comment below is not holding true:

```csharp
// Dontt ever attempt to actually change the layout of a Page unless it is a ContentPage
// iOS is a really big fan of you not actually modifying the View's of the UIViewControllers
```